packages/nextjs/services/database/repositories/large-grants.ts

```diff
@@ -0,0 +1,113 @@
+import { InferInsertModel, InferSelectModel, desc, eq, max } from "drizzle-orm";
+import { db } from "~~/services/database/config/postgresClient";
+import { largeGrants, largeMilestones, largeStages } from "~~/services/database/config/schema";
+
+export type GrantInsert = InferInsertModel<typeof largeGrants>;
+export type Grant = InferSelectModel<typeof largeGrants>;
+export type PublicLargeGrant = Awaited<ReturnType<typeof getPublicLargeGrants>>[number];
+
+export async function getAllLargeGrants() {
```

Comment:
do we need functions we dont use yet? maybe add for now `// note: not used` and remove later on use? so we don't have redundant ones

Reply:
I added the comment to all the functions not used. I think we will need almost all of these later. We can remove the non-used ones when we finish the implementation of the large grants.